.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create Release
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project
+        id: build_project
+        run: |
+          ./build.sh
+          echo "##[set-output name=dell;]$(ls *.iso)"
```

Comment:
instead of `dell` you need `blade-image` otherwise its not gonna find your asset to upload on line 38 & 39